src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -365,15 +365,15 @@ public static string FormatInvariant(string format, params object[] args)
                 return string.Format(CultureInfo.InvariantCulture, format, args.Select(SanitizeSecurityArtifact).ToArray());
         }
 
-        private static string SanitizeSecurityArtifact(object arg)
+        private static object SanitizeSecurityArtifact(object arg)
         {
             if (arg == null)
                 return "null";
 
-            if (arg is ISafeLogSecurityArtifact && IdentityModelEventSource.LogCompleteSecurityArtifact)
+            if (IdentityModelEventSource.LogCompleteSecurityArtifact && arg is ISafeLogSecurityArtifact)
```

Comment:
Use in-line is operator:

            if (IdentityModelEventSource.LogCompleteSecurityArtifact && arg is ISafeLogSecurityArtifact safeLogSecurityArtifact)
                return safeLogSecurityArtifact.UnsafeToString();